tests/unit/einstein/test_ceph.py

```diff
@@ -0,0 +1,162 @@
+import time
+import unittest
+import subprocess
+from ims.common import config
+
+config.load()
+
+import rados
+import rbd
+from ims.common import shell
+from ims.common import constants
+from ims.einstein import ceph
+from ims.einstein.operations import BMI
+from ims.exception import file_system_exceptions
+from ims.common.log import trace
+import logging
+
+_cfg = config.get()
+
+POOL = _cfg.fs.pool
+CEPH_IMG = "BMI_TEST_CEPH"
+TEST_NOT_EXIST_IMG_NAME = "BMI_TEST_IMG_NOT_FOUND"
+CEPH_IMG_SIZE = 1024
+RANGE_OUTSIDE_THE_IMAGE = 1025
+CEPH_SNAP_IMG = "BMI_TEST_CEPH_SNAP"
+TEST_DATA = 'bmi' * 10
+OFFSET = 0
+CEPH_CHILD_IMG = "BMI_TEST_CEPH_CHILD_IMAGE"
+
+
+class TestImage(unittest.TestCase):
+    """Test image operations"""
+    @trace
+    def setUp(self):
+        self.fs = ceph.RBD(_cfg.fs, _cfg.iscsi.password)
+        self.fs.create_image(CEPH_IMG, CEPH_IMG_SIZE)
+
+    def test_get_image(self):
+        """Test whether the image with given name is found or not"""
```

Comment:
Ok I checked PEP 257, there is no explicit rule that there should be a space at starting and ending of one line doc strings. Since I added space at starting and ending of doc strings at other places, better stick to it for consistency.